From 795dd90ae361fa9a4be9037911630a961fd42ef8 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Fri, 8 Feb 2019 03:05:21 +0100
Subject: [PATCH 0968/1051] spi: spi-cavium: fix compilation warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a nasty unused-variable warning during build of the spi-cavium driver:
drivers/spi/spi-cavium.c: In function ‘octeon_spi_do_transfer’:
drivers/spi/spi-cavium.c:57:6: warning: unused variable ‘enax’ \
						[-Wunused-variable]
  u64 enax = 1ull << (12 + cs);
      ^~~~

Fixes: aed6f0517ffc ("spi-cavium: spi-mux gpio pin extends to 8 chip-selects")
Change-Id: I9f52a18f5cad95bce3c5a607cedf0e9c42e205a2
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/3691
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/spi/spi-cavium.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/spi/spi-cavium.c b/drivers/spi/spi-cavium.c
index e05a7fb70cd6..ca873156ff94 100644
--- a/drivers/spi/spi-cavium.c
+++ b/drivers/spi/spi-cavium.c
@@ -54,7 +54,6 @@ static int octeon_spi_do_transfer(struct octeon_spi *p,
 	bool cpha, cpol;
 	const u8 *tx_buf;
 	u8 *rx_buf;
-	u64 enax = 1ull << (12 + cs);
 	int len;
 	int i;
 
-- 
2.17.1

